### ISSUE TITLE
Finish draw laser and draw mirror methods

### ISSUE THREAD

**Issue by elio303** [elio303/simpleGame#4]

**Comment by elio303** [comment]
Laser is done but can be improved.

**Comment by AlexDar** [comment]
Are there any benefits to using <code>context.rect()</code> instead of <code>HTML5 Canvas Lines</code>?

I'm guessing lines are better for performance due to having less properties and less to render for the browser; also shorter code.

**Comment by elio303** [comment]
Oh, you mean very thick lines? That makes sense. Maybe we should switch to that. Do mirrors look better curved? Do you mean a circular mirror? At this point, the artistic side is more flexible.

**Comment by AlexDar** [comment]
Yup, thick lines. I think straight-line mirrors would best since calculating the laser reflection angle would be easier as opposed to with a concave or convex mirror.

**Comment by elio303** [comment]
Great, agreed then. I'll work on the laser for now.

**Comment by elio303** [comment]
Laser class is done. Mirror class In Progress
